runtime/bcutil/ConstantPoolMap.cpp

```diff
@@ -439,23 +327,34 @@ ConstantPoolMap::findVarHandleMethodRefs()
 
 	for (U_16 i = 1; i < _romConstantPoolCount; i++) {
 		if ((J9CPTYPE_INSTANCE_METHOD == _romConstantPoolTypes[i])
-		|| (J9CPTYPE_INTERFACE_INSTANCE_METHOD == _romConstantPoolTypes[i])
+			|| (J9CPTYPE_INTERFACE_INSTANCE_METHOD == _romConstantPoolTypes[i])
```

Comment:
Don't change this =)  The change isn't required and the original formatting kept the if and its conditions lined up with the opening brace.  Easier to track with your eye then indenting.

```suggestion
		|| (J9CPTYPE_INTERFACE_INSTANCE_METHOD == _romConstantPoolTypes[i])
```